CauldronMods/Controller/Environments/CatchwaterHarbor/Cards/RadioPlazaCardController.cs

```diff
@@ -20,6 +20,20 @@ public override void AddStartOfGameTriggers()
             BuildTopDeckSpecialStrings();
         }
 
+        private IEnumerator FlipOverCards()
+        {
+            var decks = FindLocationsWhere(l => l.IsDeck && l.BattleZone == BattleZone);
```

Comment:
Wondering if this should have an added condition of 

 `GameController.IsTurnTakerVisibleToCardSource(l.OwnerTurnTaker, GetCardSource())` for cases of other things changing visibility? There's no location specific, but I think if an owner is not visible the deck shouldn't be either - interested to hear your thoughts on that though

Reply:
Good call. I can't find any effect that would be relevant here, so not sure I can write a unit test, but would be easy for someone to do it. I'll add the code.

Reply:
Looks good. I'll merge and include it in the next release.